Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreter.cpp

```diff
@@ -169,6 +169,18 @@ static bool CheckDSI(u32 data)
   return false;
 }
 
+static bool CheckBreakpoint(u32 data)
+{
+  PC = data;
+  PowerPC::CheckBreakPoints();
+  if (CPU::GetState() != CPU::State::Running)
+  {
+    PowerPC::ppcState.downcount = 0;
```

Comment:
I fear it might be better to update the main loop instead of forcing downcount to zero here. I'll do so later